openmp/libomptarget/test/offloading/fortran/constant-arr-index.f90

```diff
@@ -19,8 +19,8 @@ program main
      sp(5) = 10
   !$omp end target
 
-   ! print *, sp(1)
-   ! print *, sp(5)
+   print *, sp(1)
+   print *, sp(5)
```

Comment:
Woops, thank you for catching this! 

Would it be possible to add a variation of one of the other tests to use a do loop? Or just convert one of the existing ones to use a do loop, instead of the awkward while loop that was in place to work around lack of implicit capture previously!

Reply:
I've updated the test `openmp/libomptarget/test/offloading/fortran/basic-target-region-array.f90` and changed it to `target-region-implicit-array.f90` instead.